router-tests/integration_test.go

```diff
@@ -587,6 +588,128 @@ func TestVariables(t *testing.T) {
 	})
 }
 
+func TestPropagateOperationName(t *testing.T) {
+	t.Parallel()
+
+	t.Run("simple", func(t *testing.T) {
```

Comment:
I think it's required to call t.Parallel() in subtests as well to make them run in parallel.